<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_pw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PW</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_taskid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Task_ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pw">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taskid">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2018</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy.MM.dd</string>
   </property>
   <property name="date">
    <date>
     <year>2018</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>751</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_taskid_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Invisible</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_go_sheet">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Sheet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tot_count">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>총 개수:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_url">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>671</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_url">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Url</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
